### ISSUE TITLE
Incompatibility with Tom's Simple Storage Mod

### ISSUE THREAD

**Issue by Blightbuster** [Fabricators-of-Create/Create#312]
### Describe the Bug

Game crashes when starting a cannon with a schematic which uses painted blocks from [Tom's Simple Storage Mod](https://www.curseforge.com/minecraft/mc-mods/toms-storage).

```
[14:35:50] [Server thread/ERROR]: Encountered an unexpected exception
net.minecraft.class_148: Ticking block entity
	at net.minecraft.server.MinecraftServer.method_3813(MinecraftServer.java:911) ~[client-intermediary.jar:?]
	at net.minecraft.server.MinecraftServer.method_3748(MinecraftServer.java:851) ~[client-intermediary.jar:?]
	at net.minecraft.class_1132.method_3748(class_1132.java:97) ~[client-intermediary.jar:?]
	at net.minecraft.server.MinecraftServer.handler$zld000$modifiedRunLoop(MinecraftServer.java:3396) ~[client-intermediary.jar:?]
	at net.minecraft.server.MinecraftServer.method_29741(MinecraftServer.java:676) ~[client-intermediary.jar:?]
	at net.minecraft.server.MinecraftServer.method_29739(MinecraftServer.java:273) ~[client-intermediary.jar:?]
	at java.lang.Thread.run(Unknown Source) [?:?]
Caused by: java.lang.ClassCastException: class com.simibubi.create.content.schematics.SchematicWorld cannot be cast to class net.minecraft.class_3218 (com.simibubi.create.content.schematics.SchematicWorld and net.minecraft.class_3218 are in unnamed module of loader net.fabricmc.loader.impl.launch.knot.KnotClassLoader @1cd629b3)
	at com.tom.storagemod.tile.TileEntityPainted.markDirtyClient(TileEntityPainted.java:56) ~[toms_storage_fabric-1.18.2-1.3.1.jar:?]
	at com.tom.storagemod.tile.TileEntityPainted.method_11014(TileEntityPainted.java:39) ~[toms_storage_fabric-1.18.2-1.3.1.jar:?]
	at net.minecraft.class_3499.method_15172(class_3499.java:262) ~[client-intermediary.jar:?]
	at com.simibubi.create.content.schematics.SchematicPrinter.loadSchematic(SchematicPrinter.java:95) ~[create-mc1.18.2_v0.4.1+576.jar:?]
	at com.simibubi.create.content.schematics.block.SchematicannonTileEntity.initializePrinter(SchematicannonTileEntity.java:454) ~[create-mc1.18.2_v0.4.1+576.jar:?]
	at com.simibubi.create.content.schematics.block.SchematicannonTileEntity.tickPrinter(SchematicannonTileEntity.java:331) ~[create-mc1.18.2_v0.4.1+576.jar:?]
	at com.simibubi.create.content.schematics.block.SchematicannonTileEntity.tick(SchematicannonTileEntity.java:291) ~[create-mc1.18.2_v0.4.1+576.jar:?]
	at com.simibubi.create.foundation.tileEntity.SmartTileEntityTicker.tick(SmartTileEntityTicker.java:15) ~[create-mc1.18.2_v0.4.1+576.jar:?]
	at net.minecraft.class_2818$class_5563.redirect$zlg000$checkProcessTEs(class_2818.java:783) ~[client-intermediary.jar:?]
	at net.minecraft.class_2818$class_5563.method_31703(class_2818.java:659) ~[client-intermediary.jar:?]
	at net.minecraft.class_2818$class_5564.method_31703(class_2818.java:713) ~[client-intermediary.jar:?]
	at net.minecraft.class_1937.method_18471(class_1937.java:475) ~[client-intermediary.jar:?]
	at net.minecraft.class_3218.method_18765(class_3218.java:368) ~[client-intermediary.jar:?]
	at net.minecraft.server.MinecraftServer.redirect$bch000$fixUpdateSuppressionCrashTick(MinecraftServer.java:4290) ~[client-intermediary.jar:?]
	at net.minecraft.server.MinecraftServer.method_3813(MinecraftServer.java:907) ~[client-intermediary.jar:?]
	... 6 more
```

### Reproduction Steps

-

### Expected Result

-

### Operating System

Windows 10

### Mod Version

0.4.1

### Minecraft Version

1.18.2

**Comment by TropheusJ** [comment]
Can’t fix on our side, issue made: https://github.com/tom5454/Toms-Storage/issues/103

**Comment by Blightbuster** [comment]
Fixed in v1.3.2 of Tom's Simple Storage Mod
